ruleset.xml

```diff
@@ -106,8 +106,6 @@
             <property name="fixable" type="boolean" value="true"/>
         </properties>
     </rule>
-    <!-- Forbid dead code -->
-    <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements"/>
```

Comment:
No longer a thing.  [It's handled by PHPStan now](https://phpstan.org/blog/detecting-unused-private-properties-methods-constants).